src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -49,6 +50,8 @@
     "QuantizationModifier",
 ]
 
+Mod2Quant = collections.namedtuple("Mod2Quant", ["name", "module"])
```

Comment:
@natuan for consistency, please change over to NamedTuple from typings. Additionally, we need to add types for the attributes in this collection. Finally, if this is meant to be a private, internal variable please mark it as such by prefacing with an underscore: https://github.com/neuralmagic/sparseml/blob/a574de1dbac844080e986783ef7c88858c9a3a29/src/sparseml/onnx/utils/helpers.py#L226

Also, I agree with Ben that we want to stay away from putting numbers in our variable names. We've kept names verbose and clean for our current implementations and don't want to move away from that

Reply:
Changed to _ModuleToQuantize